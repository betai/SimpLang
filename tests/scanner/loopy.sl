    let a = 1 and
		loopy = a+-1
	in
		loopy
	end
