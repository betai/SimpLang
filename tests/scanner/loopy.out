keyword let
identifier a
operator =
integer 1
keyword and
identifier loopy
operator =
identifier a
operator +
operator -
integer 1
keyword in
identifier loopy
keyword end
